api/src/main/java/org/openmrs/module/idgen/service/db/HibernateIdentifierSourceDAO.java

```diff
@@ -72,7 +72,7 @@ public IdentifierSource getIdentifierSource(Integer id) throws APIException {
 	public List<IdentifierSource> getAllIdentifierSources(boolean includeRetired) throws DAOException {
 		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(IdentifierSource.class);
 		if (!includeRetired) {
-			criteria.add(Expression.like("retired", false));
```

Comment:
Is this related?

Reply:
Yes with expression.like you get an error raised.